applications/serial_lte_modem/src/main.c

```diff
@@ -156,9 +158,21 @@ void handle_bsdlib_init_ret(void)
 void start_execute(void)
 {
 	int err;
+	struct onoff_manager *clk_mgr;
+	struct onoff_client cli;
```

Comment:
According to docs, the `node` field of this structure should be zero-initialized by the user, it would be easiest just to clear the structure at this point `struct onoff_client cli = {};`
Reference: https://github.com/zephyrproject-rtos/zephyr/blob/master/include/sys/onoff.h#L259

Reply:
@rlubos Thanks for the review comment.
Change applied